website/source/guides/web-ui/inspecting-the-cluster.html.md

```diff
@@ -0,0 +1,150 @@
+
+### Resource Utilization
+
+Nomad as APIs for reading point-in-time resource utilization metrics for client nodes. The Web UI
+uses these metrics to create time-series graphics for the current session.
+
+When viewing a client node, resource utilization will automatically start logging.
+
+[![Client Resource Utilization][img-client-resource-utilization]][img-client-resource-utilization]
+
+### Allocations
+
+Allocations belong to jobs and are placed on client nodes. The Client Detail page will list all
+allocations for a client node, including completed, failed, and lost allocations, until they are
+garbage-collected.
+
+This is presented in a searchable table which can additionally be filtered to only preempted
+allocations.
+
+[![Client Allocations][img-client-allocations]][img-client-allocations]
+
+### Client Events
+
+Client nodes will also emit events on meaningful state changes, such as when the node becomes ready
+for scheduling or when a driver becomes unhealthy.
+
+[![Client Events][img-client-events]][img-client-events]
+
+### Driver Status
+
+Task drivers are additional services running on a client node. Nomad will fingerprint and
+communicate with the task driver to determine if the driver is available and healthy. This
+information is reported through the Web UI on the Client Detail page.
+
+[![Client Driver Status][img-client-driver-status]][img-client-driver-status]
+
+### Attributes
+
+In order to allow job authors to constrain the placement of their jobs, Nomad fingerprints the
+hardware of the node the client agent is running on. This is a deeply nested document of properties
+that the Web UI presents in a scannable way.
+
+In addition to the hardware attributes, Nomad operators can annotate
+[a client node with metadata](/docs/configuration/client.html#meta) as part of the client configuration. This metadata
+is also presented on the Client Detail page.
+
+[![Client Attributes][img-client-attributes]][img-client-attributes]
+
+## When a Node is Draining
+
+A routine part of maintaining a Nomad cluster is draining nodes of allocations. This can be in
+preparation of performing operating system upgrades or decommissioning an old node in favor of a new
+VM.
+
+Drains are [performed from the CLI](/guides/operations/node-draining.html) but the status of a drain
+can be seen from the Web UI. A client node will state if it is actively draining or ineligible for
+scheduling.
+
+Since drains can be configured in a variety of ways, the Client Detail page will also present the
+details of how the drain is performed.
+
+[![Client Drain][img-client-drain]][img-client-drain]
+
+## Reviewing All Servers
+
+Where client nodes are used to run your jobs, server nodes are used to run Nomad and maintain
+availability. From any page, the Servers List page can be accessed from the left-hand navbar.
+
+Here you can see every server node. This will be a small list—typically three, but maybe five or
+seven.
```

Comment:
I would stick to just mentioning 3 or 5 and maybe link to this page:

https://www.nomadproject.io/docs/internals/consensus.html#deployment-table